lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -47,6 +47,56 @@
         description:
             - Boolean indication whether to allow partial registration of Virtual Machine when C(state) is registered.
         version_added: "2.4"
+    vnic_profile_mappings:
+        description:
+            - "Mapper which maps an external virtual NIC profile to one that exists in the engine when C(state) is registered.
+               vnic_profile is described by the following dictionary:
+              - C(source_network_name): The network name of the source network.
```

Comment:
Please use right formatting as I've described above for `templates` module.

Reply:
done